lib/cc/engine/scalastyle.rb

```diff
@@ -57,22 +57,28 @@ def analysis_files
       end
 
       def scalastyle_jar
-        '/home/app/scalastyle_2.11-0.6.0-batch.jar'
+        '/home/app/scalastyle_2.11-0.8.0-batch.jar'
       end
 
       def scalastyle_config
-        @engine_config['config'] || '/home/app/scalastyle_config.xml'
+        @engine_config['config'] || '/code/scalastyle_config.xml'
```

Comment:
Scalastyle is highly configurable, so people will have their own config for each code tree.

We could provide one by default, but I would prefer to have the user select one themselves.